src/js/components/homepageUpdate/AwardSearch/AwardSearch.jsx

```diff
@@ -78,21 +81,21 @@ const AwardSearch = () => {
             version: REQUEST_VERSION
         };
 
-        if (tab === "map") {
+        if (section === "map" || section === "geography") {
```

Comment:
why do you have map or geography here for section? we need the original tab logic in addition to the section logic, otherwise this will break production